lib/rules/length-zero-no-unit/README.md

```diff
@@ -87,3 +87,25 @@ a { top: var(--foo, 0px); }
 ```css
 a { top: --bar(0px); }
 ```
+
+### `ignorePreludeOfAtRules: ["/regex/", /regex/, "string"]`
+
+Ignore units for zero lengths within the preludes of the specified at-rules.
+
+Given:
+
+```json
+["--foo", "/^--bar/"]
+```
+
+The following patterns are _not_ considered problems:
+
+<!-- prettier-ignore -->
+```css
+@--foo 0px {}
+```
+
+<!-- prettier-ignore -->
+```css
+@--bar-baz 0px;
+```
```

Comment:
[suggestion] Would you be willing to add a more practical example? E.g.

```json
"ignorePreludeOfAtRules": ["media"]
```

```css
@media (height > 0px) {}
```

Reply:
Let's replace `--foo` with `media` and keep the `^--bar` custom at-rule example to show both use cases.